src/operation/get-parameters-as-json-schema.js

```diff
@@ -483,7 +483,7 @@ function getParameters(path, operation, oas, globalDefaults) {
       let schema = {};
       if ('schema' in current) {
         schema = {
-          ...(current.schema ? constructSchema(current.schema, [], '', globalDefaults) : {}),
+          ...(current.schema ? constructSchema(current.schema, [], `/${current.name}`, globalDefaults) : {}),
```

Comment:
Might need to do the same fix on lien 508?

Reply:
Tested it with a file to make sure that line needed to be updated too and it did! Updated!